fastestimator/util/util.py

```diff
@@ -214,6 +215,37 @@ def _custom_tf_print(*args, **kwargs):
     print_v2(*args, **kwargs)
 
 
+def validate_file(file_path: str) -> bool:
+    """Validate whether file is valid or not.
+
+    Args:
+        file_path: location of the input file.
+
+    Returns:
+        Whether the file is valid.
+    """
+    try:
+        if Path(file_path).suffix == '.zip':
+            import zipfile
+            zip_file = zipfile.ZipFile(file_path)
+            _ = zip_file.namelist()
+        elif Path(file_path).suffix == '.gz':
+            if file_path.endswith('.tar.gz'):
+                import tarfile
+                with tarfile.open(file_path) as img_tar:
+                    _ = img_tar.getmembers()
+            else:
+                import gzip
+                f = gzip.open(file_path, 'rb')
+                _ = f.read()
+        elif not (os.path.exists(file_path) and os.path.getsize(file_path) > 0):
```

Comment:
why does the file size need to be >0 here? wouldn't an empty file that passed the above tests technically still be valid?